app/views/calendars/_current_events.html.erb

```diff
@@ -1,14 +1,14 @@
 <table class="table table-striped">
   <thead>
     <tr>
-      <th>Type</th>
-      <th>Start Date</th>
-      <th>End Date</th>
-      <th>Last Updated</th>
+      <th><%= t('calendars.current_events.type') %></th>
+      <th><%= t('calendars.current_events.start_date') %></th>
+      <th><%= t('calendars.current_events.end_date') %></th>
+      <th><%= t('calendars.current_events.last_updated') %></th>
       <% if can? :manage, calendar %>
-        <th>Status</th>
+        <th><%= t('calendars.current_events.status') %></th>
       <% end %>
-      <th>Actions</th>
+      <th><%= t('calendars.current_events.actions') %></th>
```

Comment:
Could use common label here?

Reply:
:+1: have pushed an update to use common labels